Paco/res/xml/runtime_permissions_accessibility_config.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Makes sure that the com.pacoapp.paco.sensors.android.RuntimePermissions
+     service will only listen for accessibility events from the packageinstaller
+     and that it can retrieve the contents on screen.
+     We pass on the flagReportViewIds flag to make it easier to identify the
+     parts of the interface that contain the information we need -->
+<accessibility-service xmlns:android="http://schemas.android.com/apk/res/android"
+    android:packageNames="com.google.android.packageinstaller,com.android.packageinstaller"
```

Comment:
If I understood correctly, OEMs are not allowed to interfere with the `packageinstaller` in any way, so this should be consistent across all vendor customizations. This is also the reason using an accessibility service would work, otherwise I agree it would be a very unstable approach :-)